static/js/components/email/AutomaticEmailOptions.js

```diff
@@ -28,15 +25,19 @@ export default class EmailCompositionType extends React.Component {
     </div>
   );
 
+  getEmailType = (sendAutomaticEmails?: boolean): string => (
+    sendAutomaticEmails ? EMAIL_CAMPAIGN : ONE_TIME_EMAIL
+  )
+
   render() {
-    const { automaticEmailType } = this.props;
+    const { sendAutomaticEmails } = this.props;
 
     return (
       <div className="email-type">
         <RadioButtonGroup
           className="type-radio-group"
           name="email-composition-type"
-          valueSelected={automaticEmailType}
+          valueSelected={this.getEmailType(sendAutomaticEmails)}
```

Comment:
get rid of `EMAIL_CAMPAIGN` and `ONE_TIME_EMAIL` completely, in other words, is my suggestion